recipes/secretstorage/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "SecretStorage" %}
+{% set version = "3.0.1" %}
+{% set sha256 = "819087ca89c0d6c5711692f41fb26f786af9dcc5bb89d567722a66edfbb2a689" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.5
+    - pip
+    - setuptools
+  run:
+    - python >=3.5
+    - jeepney
+    - cryptography
+
+test:
+  imports:
+    - secretstorage
+
+about:
+  home: https://github.com/mitya57/secretstorage
+  license: OTHER
+  license_file: LICENSE
+  summary: 'Provides a way for securely storing passwords and other secrets.'
+  description: |
+    This module provides a way for securely storing passwords and other secrets.
+    It uses D-Bus Secret Service API that is supported by GNOME Keyring (since 
+    version 2.30) and KSecretsService.
```

Comment:
This sounds like a Linux only package. If so, please reflect that in the skip statement.

Reply:
Would this be the correct way to construct the skip statement in this situation?

```yaml
build:
  skip: True  # [win or osx ]
```